packages/SwingSet/test/timer-device/bootstrap.js

```diff
@@ -25,7 +26,7 @@ export function buildRootObject(vatPowers, vatParameters) {
             );
           },
         });
-        const rptr = D(devices.timer).makeRepeater(argv[1], argv[2]);
+        const rptr = D(devices.timer).makeRepeater(Nat(argv[1]), Nat(argv[2]));
```

Comment:
What type is `argv[1]` etc? From other languages I'd expect these to be strings, in which case this won't work. But `Nat(BigInt(argv[1]))` would.

Reply:
These are Numbers. @warner is working on allowing us to pass in BigInts directly.